template/scripts/sample-script.js

```diff
@@ -13,13 +13,21 @@ async function main() {
   // manually to make sure everything is compiled
   // await hre.run('compile');
 
-  // We get the contract to deploy
-  const Greeter = await hre.ethers.getContractFactory("Greeter");
-  const greeter = await Greeter.deploy("Hello, Hardhat!");
+  // We get the contracts to deploy
+  const ERC20Token = await hre.ethers.getContractFactory('ERC20Token');
+  const token = await ERC20Token.deploy();
+  await token.deployed();
+  console.log('Token deployed to:', token.address);
 
-  await greeter.deployed();
+  const NFT = await hre.ethers.getContractFactory('NFT');
+  const nft = await NFT.deploy();
+  await nft.deployed();
+  console.log('NFT deployed to:', nft.address);
 
-  console.log("Greeter deployed to:", greeter.address);
+  const Greeter = await hre.ethers.getContractFactory('Greeter');
```

Comment:
We probably don't need the `Greeter` anymore

Reply:
I've kept it there so that the frontend doesn't break for now. Will remove it once the contracts have been integrated on the frontend.